docs/csharp/roslyn/concepts/compiler-api-model.md

```diff
@@ -0,0 +1,104 @@
+---
```

Comment:
do you expect this area to grow a lot in size? adding levels in the url is not good for seo. also roslyn is more than just c#, no? shouldn't we have a Roslyn entry on the TOC?
Otherwise, I'd advise that we have a url like docs.microsoft.com/dotnet/roslyn and the topics directly under it (unless you expect a big expansion then having concepts is fine)